nodebuilder/p2p/flags.go

```diff
@@ -2,28 +2,41 @@ package p2p
 
 import (
 	"fmt"
+	"os"
+	"strings"
 
 	"github.com/multiformats/go-multiaddr"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
 )
 
-var (
-	p2pMutualFlag = "p2p.mutual"
+const (
+	// DefaultNetwork is the default network of the current build.
+	DefaultNetwork   = Arabica
+	EnvCustomNetwork = "CELESTIA_CUSTOM"
+	networkFlag      = "p2p.network"
+	mutualFlag       = "p2p.mutual"
 )
 
 // Flags gives a set of p2p flags.
 func Flags() *flag.FlagSet {
 	flags := &flag.FlagSet{}
 
 	flags.StringSlice(
-		p2pMutualFlag,
+		mutualFlag,
 		nil,
 		`Comma-separated multiaddresses of mutual peers to keep a prioritized connection with.
 Such connection is immune to peer scoring slashing and connection manager trimming.
 Peers must bidirectionally point to each other. (Format: multiformats.io/multiaddr)
 `,
 	)
+	flags.String(
+		networkFlag,
+		"",
+		"The name of the network to connect to, e.g. "+
+			listProvidedNetworks()+
+			". Is only effective for the start command, and has no effect on init.",
```

Comment:
```suggestion
			". Must be passed on both init and start to take effect.",
```

This is because it's needed during `init` for store initialisation purposes but **must also** be passed on `start`

Reply:
done